### PR TITLE
I18n setup + shared/footer I18n

### PR DESCRIPTION
**Pull request by jiteshjha**
Hi @tarebyte, I've removed the test and carybara.

Thank you for your input!

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -53,6 +53,8 @@ gem 'turbolinks', '~> 2.5.3'
 
 gem 'uglifier', '>= 1.3.0'
 
+gem 'rails-i18n', '~> 4.0.0'
```
Comment (cyhsutw) [inline Gemfile]: Hi @jiteshjha 👋 

One minor suggestion, I think the Gems are ordered in alphabetical order (except for the `rails` gem). 

What do you think about inserting this line like this:

```diff
  gem 'rack-canonical-host'
  gem 'rack-timeout', require: false
+ gem 'rails-i18n', '~> 4.0.0'
  gem 'redis-namespace'
```

Thanks!

Comment (jiteshjha) [inline Gemfile]: Oh! I didn't know that. 
Thanks!

Comment (cyhsutw) [inline Gemfile]: No problem, it's just too subtle to be noticed 😉

**Comment by tarebyte** [comment]
Thanks for the PR @jiteshjha! I'd be more than happy to merge this if you could merge master into your branch. Thanks!

**Comment by jiteshjha** [comment]
Hi! @tarebyte I have rebased it over the latest master.
Thanks! :)

**Pull request merged**